packages/cli/src/commands/index.ts

```diff
@@ -365,6 +365,7 @@ export function registerCommands(program: Command) {
       '--format <format>',
       'Format to print the schema in, either json or yaml [yaml]',
     )
+    .option('--non-merged', 'Enable to print the config schema non merged')
```

Comment:
Enabling/disabling flags are commonly named `--foo` / `--no-foo`.

https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue

This one should be named `--no-merge`, and then you read `opts.merge`

Reply:
Gotcha, updated it